CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ## Features
 
 - Add support for `get_default_subnets` to `@dfinity/cmc`.
+- Add class `AgentManager` in `@dfinity/utils` which scope is to manages HttpAgent instances for different identities.
```

Comment:
Suggestion:

- Add class `AgentManager` in `@dfinity/utils` which caches HttpAgent instances for different identities.

Reply:
changed